tools/ci_build/github/linux/ort_nnapi_minimal/nnapi_minimal_build_full_ort_and_create_ort_files.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# This script will run a full ORT build and use the python package built to generate ort format test files,
+# and the exclude ops config file, which will be used in the build_minimal_ort_and_run_tests.sh
+
```

Comment:
Are we duplicating too much stuff here vs. adding to the Android CI? Won't that just add more work to the CI servers, so it's better to have one longer CI than to add another one that duplicates some of the work?

Reply:
Move the minimal build and run UT to Android CI
Will add some onnx_test_runner case later